static/templates/admin_user_group_list.hbs

```diff
@@ -12,6 +12,7 @@
         <button class="button save-status btn-danger small">
             {{t 'Discard changes' }}
         </button>
+        <button class="save-discard-widget-button save-status button primary save-button" id='save-user-group'>save</button>
```

Comment:
While this certainly works, the styling of this is not the same as other `save` buttons in the user_group page.